gateway/src/api/project_caller.rs

```diff
@@ -34,7 +34,10 @@ impl ProjectCaller {
             service, sender, ..
         } = state;
         let project_name = scoped_user.scope;
-        let project = service.find_or_start_project(&project_name, sender).await?;
+        let project = service
+            .find_or_start_project(&project_name, sender)
+            .await?
+            .0;
```

Comment:
We then need to use this directly when creating `Self {` below, right?

Reply:
no... ? wdym?

Reply:
Oh, sorry. Missed the change you did in `find_or_start_project`.